DB/src/main/java/io/deephaven/db/v2/locations/impl/TableLocationFactory.java

```diff
@@ -23,5 +23,5 @@
      *                       {@link TableLocationProvider#supportsSubscriptions() supports subscriptions}
      * @return A new or cached {@link TableLocation} identified by the supplied {@link TableKey} and {@link TableLocationKey}
      */
-    TableLocation makeLocation(@NotNull final TK tableKey, @NotNull TLK locationKey, @Nullable TableDataRefreshService refreshService);
+    TableLocation makeLocation(@NotNull TK tableKey, @NotNull TLK locationKey, @Nullable TableDataRefreshService refreshService);
```

Comment:
final dropped but no later change on the file?  guessing you may have been changing the code and needing mutating the argument but then that wasn't the case in the final version?

Reply:
I was looking at this code for some reason, and realized I had `final` declared on an interface method (which is silly/useless) so got rid of it.